scripts/mysql-vuln-cve2017-3599.nse

```diff
@@ -0,0 +1,69 @@
+description=[[
```

Comment:
The order of code isn't correct.
```
local vulns = require "vulns"
description = [[]]
-- @usage
-- @output
```
Follow this order in your code or checkout other scripts for reference purpose.